docs/SNMP_Alarms.md

```diff
@@ -41,3 +41,13 @@ obtain the brief description, detailed description, and severity for the alarm:
 ## Alarm Models
 
 The alarm models used by Clearwater are defined in [alarmdefinition.h](https://github.com/Metaswitch/cpp-common/blob/master/include/alarmdefinition.h).
+
+## Alarm Resiliency
+
+Alarms are reported in such a way that if a component receiving alarm
```

Comment:
What is "a component receiving alarm notifications"? If you mean the component we've referred to above as an EMS (Element Management System - same thing as NMS, or Network Management System) then you should say EMS here too. "[Express co-ordinate ideas in similar form](https://faculty.washington.edu/heagerty/Courses/b572/public/StrunkWhite.pdf)"!